UnityProject/Assets/Scripts/UI/Vending/GUI_Spawner.cs

```diff
@@ -0,0 +1,222 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class GUI_Spawner : NetTab
+{
+	private ItemList entryList;
+	private ItemList PrefabEntryList => entryList ? entryList : entryList = this["EntryList"] as ItemList;
+
+	private SpawnedObjectList spawnedObjectList;
+	private SpawnedObjectList SpawnedObjectList => spawnedObjectList ? spawnedObjectList : spawnedObjectList = this["MobList"] as SpawnedObjectList;
+
+	private NetUIElement<string> infoDisplay;
+	private NetUIElement<string> InfoDisplay => infoDisplay ? infoDisplay : infoDisplay = (NetUIElement<string>)this["RandomText"];
```

Comment:
Codacy found an issue: [Extract the assignment of 'infoDisplay' from this expression.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6280689)